.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules/
 .nyc_output
 coverage
 migrations/
+.vscode
```

Comment:
Not in my case, I also have a language server project which includes writing a custom vscode extension and we do commit the `.vscode` dir with configurations for debugging. This may look like an exceptional case, so I'd rather remove this change and just not commit the `.vscode` dir.

Reply:
Done in https://github.com/okv/east/pull/54/commits/77a3c0fd31e73eb2b839d0485e0511ca5ac73bcf